src/components/Select.js

```diff
@@ -0,0 +1,136 @@
+import app from '../firebase';
+import { onValue, getDatabase, ref, get } from 'firebase/database'
+import { useState, useEffect } from 'react';
+
+
+const Select = () => {
+
+  const database = getDatabase(app);
+
+  const [ allSurfaces, setAllSurfaces ] = useState([]);
+  const [ allSkaterPref, setAllSkaterPref ] = useState([]);
+  const [ selectedSurface, setSelectedSurface ] = useState('');
+  const [ selectedSkaterPref, setSelectedSkaterPref ] = useState('');
+  const [ userSelection, setUserSelection ] = useState([]);
+  //const [ range, setRange ] = useState([""]);
+  
+
+
+
+  useEffect( () => {
+    const database = getDatabase(app);
+    const surfaceRef = ref(database, '/surfaceType');
+    
+      onValue(surfaceRef, (dbResponse) => {
+        const surfaceList = Object.keys(dbResponse.val());
+        setAllSurfaces(surfaceList);
+  
+      });
+  
+    }, []);
+  
+    useEffect(() => {
+  
+      if (selectedSurface) {
+        const database = getDatabase(app);
+        const skaterPrefRef = ref(database, `surfaceType/${selectedSurface}/skaterPreference`);
+  
+        onValue(skaterPrefRef, (dbResponse) => {
+          const skaterPrefList = Object.keys(dbResponse.val());
+  
+          setAllSkaterPref(skaterPrefList);
+  
+        });
+  
+      } else {
+        setAllSkaterPref([]);
+      }
+  
+      }, [selectedSurface]);
+  
+
+
+
+  const handleSurfaceChange = (event) => {
+    setSelectedSurface(event.target.value);
+  };
+
+  const handleSkaterPrefChange = (event) => {
+    setSelectedSkaterPref(event.target.value);
+  };
+
+
+
+
+  const handleUserSubmit = (event) => {
+    event.preventDefault();
+
+
+    const userSelection = ref(database, `surfaceType/${selectedSurface}/skaterPreference/${selectedSkaterPref}`);
+
+    get(userSelection).then (results => {
+      setUserSelection(results.val())
+     
+    })
+
+  }
+
+
+  return (
+  
+<section class="selector">
+
+  <form value={userSelection} onSubmit={handleUserSubmit}>
+        <div className="formMain">
+        <label>Select:
```

Comment:
Maybe use more descriptive label like Select Surface?  Also don't forget for and id for accessibility.